src/Phinx/Console/Command/AbstractCommand.php

```diff
@@ -52,6 +52,26 @@ abstract class AbstractCommand extends Command
      */
     protected $manager;
 
+    /**
+     * @var int
+     */
+    const EXIT_SUCCESS = 0;
```

Comment:
Should we use EXIT_ or CODE_? Refs https://github.com/cakephp/migrations/blob/master/src/Command/BaseCommand.php

Technically, it is a (status) code to be returned into CLI when existing, right?

CODE_SUCCESS
CODE_ERROR
CODE_MISSING
CODE_DOWN

?

Reply:
And better to explain them a bit maybe in the docblock

Reply:
Ah, I did not realize there was an existing standard in Cake, I'll update it to:
* CODE_SUCCESS
* CODE_ERROR
* CODE_STATUS_MISSING
* CODE_STATUS_DOWN